website/docs/r/detective_invitation_accepter.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Detective"
+layout: "aws"
+page_title: "AWS: aws_detective_invitation_accepter"
+description: |-
+  Provides a resource to manage an Amazon Detective Invitation Accepter.
+---
+
+# Resource: aws_detective_invitation_accepter
+
+Provides a resource to manage an [Amazon Detective Invitation Accepter](https://docs.aws.amazon.com/detective/latest/APIReference/API_AcceptInvitation.html).
+
+## Example Usage
+
+```terraform
+resource "aws_detective_graph" "primary" {
+  provider = "awsalternate"
+}
+
+resource "aws_detective2_member" "primary" {
+  provider   = "awsalternate"
+  account_id = "ACCOUNT ID"
+  email      = "EMAIL"
+  graph_arn  = aws_detective_graph.admin.id
+  message    = "Message of the invite"
+}
+
+resource "aws_detective_invitation_accepter" "member" {
+  graph_arn = aws_detective_graph.primary.id
+}
```

Comment:
I especially don't think it makes sense to use the reverse provider setup here in docs.

The example here ought to have only `aws_detective_invitation_accepter` provisioned by `provider = "awsalternate"`.